crates/transaction-pool/src/validate/eth.rs

```diff
@@ -274,6 +284,8 @@ struct EthTransactionValidatorInner<Client, T> {
     minimum_priority_fee: Option<u128>,
     /// Marker for the transaction type
     _marker: PhantomData<T>,
+    /// Toggle to determine if a local transaction should be propagated
+    local_tx_propagation: bool,
```

Comment:
I'd like to rename this
```suggestion
    /// Toggle to determine if a local transaction should be propagated
    propagate_local_transactions: bool,
```